### ISSUE TITLE
[Bug]: Sending a social club message via yimmenu indicating success did not actually succeed!

### ISSUE THREAD

**Issue by ltyes** [Yimura/YimMenu#2051]
### Describe the bug

[Bug]: Sending a social club message via yimmenu indicating success did not actually succeed!

### Steps To Reproduce

![image](https://github.com/YimMenu/YimMenu/assets/143527816/ade603d7-7bb8-44db-b6bb-8f56b0366e2d)
![image](https://github.com/YimMenu/YimMenu/assets/143527816/69f29d6b-162c-4e79-8688-4c6c5355a9d9)

### Expected Behavior

Make the yimmenu menu send club messages normally. I think it's very convenient!

### Logs and/or screenshots

```
<paste your logs here>
```

### Store

Steam

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### Coload

- [X] I've tested this without co-loading anything.

**Comment by spinoza-spinoza** [comment]
yes i confirmed  ..the  message sent doesnt appear in social club  ..i tested 1 month ago

**Comment by lonelybud** [comment]
i can confirm this.
